packages/mux-player/src/video-api.ts

```diff
@@ -129,20 +129,27 @@ class VideoApiElement extends HTMLElement {
     options?: boolean | EventListenerOptions
   ) {
     if (AllowedVideoEvents.includes(type)) {
-      this.video?.removeEventListener(type, listener, options);
+      this.media?.removeEventListener(type, listener, options);
       return;
     }
     super.removeEventListener(type, listener, options);
   }
 
   play() {
-    return this.video?.play();
+    return this.media?.play();
   }
 
   pause() {
-    this.video?.pause();
+    this.media?.pause();
   }
 
+  get media(): MuxVideoElement | null | undefined {
+    return this.shadowRoot?.querySelector('mux-video');
+  }
+
+  /**
+   * keep .video for backwards compatibility.
+   */
   get video(): MuxVideoElement | null | undefined {
     return this.shadowRoot?.querySelector('mux-video');
   }
```

Comment:
Should we keep this? If so, should we log a warning encouraging folks to move over?